File: Php/test-client.php
<?php
	require_once("JSON-RPC-Stomp-Client.php");
	error_reporting(E_ERROR | E_WARNING | E_PARSE);
	//error_reporting(E_ALL);
	if(!json_rpc_stomp_init("tcp://localhost:61613"))
		die("Error: $json_rpc_error ($json_rpc_errormsg ${json_rpc_errormsgdefine[$json_rpc_error]})\n");
$start = time();
for ($i = 0; $i < 1; $i++) {
	$ret = json_rpc_call("jsonrpc","subtract",
		array($i,5) );
	if ($ret == null) {
		print "Error: $json_rpc_error ($json_rpc_errormsg ${json_rpc_errormsgdefine[$json_rpc_error]})\n";
	}
	else {
		print "Result:\n";
		var_dump($ret);
	}
}
$stop = time();
echo "Run time: ".($stop-$start)."\n";
?>

File: Perl/test-client.pl
#!/usr/bin/perl
use strict;
use JSON::RPC::Simple::Common;
use JSON::RPC::Simple::StompClient;

json_rpc_stomp_init('localhost:61613');

for (1..2) {
my $ret = json_rpc_call('jsonrpc','subtract',[52,49]);

print "$json_rpc_error $json_rpc_errormsgdefine{$json_rpc_error} $json_rpc_errormsg\n" unless ($ret);
print "$ret->{result}, call-id:$ret->{id}\n";
}
